### PR TITLE
The end of Pubbystation (and Glowstation aswell)

### PR DESCRIPTION
**Pull request by PestoVerde322**
## About The Pull Request

This PR just removes Pubbystation and Glowstation from the code, after a prolonged discussion within the dev group.

Also expunged as many reference from pubby as possible, Aside for the shuttles, no latient content that refeer to something that no longer exist.

## Why It's Good For The Game

First and foremost, this PR is being done due to streamline the process on updating maps if any sort of gameplay update happens (remember when almost all of the viros had to be updated?)

Second off, Pubby is a poorly maintained map, with an even poorer layout; while Glow is kinda underbaked as it stands right now (may come in a different form if kat decides to make a better, more stable version).

Lastly, It's for the mental sake of all the map maintainers to just work around maps better and give room to more home built maps like Radstation.

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>

N/A, what i should show if it's just gone?

It's the end of Pubbystation as we know it, and i feel fine (also current glow, but not for long hopefully).

</details>

## Changelog
:cl:
del: Removed Pubby station off the rotation
del: Removed Glow station off the rotation
/:cl:

### CODE REVIEW

**Review by imsynthex — APPROVED** [review]
I don't know whose idea it was but I approve

**Comment by PowerfulBacon** [comment]
_maps/shuttles/exploration/exploration_fland.dmm:197:error: undefined type: /obj/machinery/vending/boozeomat/pubby_maint
_maps/shuttles/exploration/exploration_fland.dmm:200:error: unknown type

**Comment by Crossedfall** [comment]
This seems wrong
image

**Review by Crossedfall — APPROVED** [review]
The booze-o-mat shows up on local (so I've been told). I'm good with the changes. This will need wiki updates.

**Comment by Tsar-Salat** [comment]
> This will need wiki updates.

Recommend the creation of a new legacy maps page for strictly-Beestation maps. There we can put Glow and Pubby's pages in their entirety. This will let us easily replace Glow's wikipage again if someone decides to readd it.

**Review by PowerfulBacon — APPROVED** [review]
This will be merged later, upon the merging of RadStation

**Comment by itsmeow** [comment]
This also should update the README which has a map list

File: code/modules/vending/medical_wall.dm [hunk 1]
```diff
@@ -26,7 +26,9 @@
 	machine_name = "NanoMed"
 	icon_state = "refill_medical"
 
-/obj/machinery/vending/wallmed/pubby
+/obj/machinery/vending/wallmed/Lite
```
Comment (itsmeow) [inline code/modules/vending/medical_wall.dm]: don't capitalize typepaths

File: _maps/map_files/CorgStation/CorgStation.dmm [hunk 2]
```diff
@@ -31495,7 +31495,7 @@
 /turf/open/floor/plasteel,
 /area/hallway/primary/central)
 "kkg" = (
-/obj/machinery/vending/boozeomat/pubby_captain,
+/obj/machinery/vending/boozeomat/captain ,
```
Comment (itsmeow) [inline _maps/map_files/CorgStation/CorgStation.dmm:31498]: space here

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Comment by PowerfulBacon** [comment]
how is the readme conflicting 💀

File: code/modules/vending/boozeomat.dm [hunk 3]
```diff
@@ -66,7 +74,8 @@
 			/obj/item/reagent_containers/food/drinks/flask = 1)
 	req_access = null
 
-/obj/machinery/vending/boozeomat/pubby_captain //Captain's quarters on Pubbystation
+/obj/machinery/vending/boozeomat/captain//Captain's quarters variant
```
Comment (Crossedfall) [inline code/modules/vending/boozeomat.dm:77]:
```suggestion
/obj/machinery/vending/boozeomat/captain //Captain's quarters variant
```

I actually prefer comments above the object, but, this keeps it consistent.

**Review by itsmeow — APPROVED** [review]

**Review by Crossedfall — APPROVED** [review]

**Pull request merged**